### DISCUSSION TITLE
Code-server 3.12 not displaying terminal properly

### DISCUSSION THREAD

**Discussion by spideysri2802** [coder/code-server#4334] [Q&A] [answered]
Just upgraded from code-server 3.11 to 3.12, and now terminal isn't displaying text.
Anybody know how to fix this? Any help is much appreciated, thank you! 

<img width="441" alt="Screenshot 2021-10-10 at 4 39 47 PM" src="https://user-images.githubusercontent.com/17454186/136688788-f477556b-4e6b-43cf-84cf-3cfb79c245f6.png">

**Comment by jsjoeio** [comment, marked as answer]
Hmmm..... are there any server or browser logs? What happens if you close and reopen the terminal? Are you using a custom font?

There's so many things that can go wrong here, it's hard to offer one solution. But here's a list of things to try:
- check browser & server logs
- restart code-server, browser and computer
- refresh page
- disable all extensions
- reopen terminal window

**Reply by spideysri2802** [reply]
opening and closing the terminal doesnt fix it. i even tried specifying background and foreground color, but the text still remains invisible. 
have restarted code-server, even reinstalled it completely.. it works on my pc, just doesnt seem to work on my macbook. could safari be the problem?

**Reply by lg-nineoneninethreenine** [reply]
Well, judging by Safari, have you tried Chromium-based browsers like Chrome, Brave, Vivaldi or Chromium itself? Maybe it is just that Safari is preventing an important cookie to load up.

**Reply by lg-nineoneninethreenine** [reply]
Or is JavaScript enabled for Safari?

**Reply by spideysri2802** [reply]
the weird thing is. it was working fine even on safari until i updated to code-server 3.12. and yes, i just tried it on chrome, and it seems to work fine. maybe it is a safari problem. yes, js is enabled in safari.

**Reply by anikaly10** [reply]
It worked for me when I turn off Integrated GPU acceleration in Terminal section of Code Server settings, hope this helps...

**Reply by jsjoeio** [reply]
Are there any browser logs? Can you try using a different vscode theme?

**Reply by spideysri2802** [reply]
> Are there any browser logs? Can you try using a different vscode theme?

nope nothing.. tried using a different theme, terminal still doesnt show up. sometimes it shows for a second and then disappears.

**Reply by spideysri2802** [reply]
> It worked for me when I turn off Integrated GPU acceleration in Terminal section of Code Server settings, hope this helps...

hi, thank you so much! this fixed it!
